pkg/validator/types.go

```diff
@@ -45,6 +45,12 @@ type CountSummary struct {
 	Errors    uint
 }
 
+// GetScore returns an overall score in [0, 100] for the CountSummary
+func (cs *CountSummary) GetScore() uint {
```

Comment:
Is there documentation around this weighting process? (I.e. warnings decrease your score 1/2 as much as errors)? (non-blocking)

Reply:
It's mentioned on the dashboard itself - not sure if you'd call that "documentation" :)

Reply:
oh no worries - not a huge deal - :) thanks!